### ISSUE TITLE
Implement all resamplers in scikit

### ISSUE THREAD

**Issue by jrnold** [jrnold/resamplr#1]
Implement resampler versions of all cross validation methods in [sklearn](http://scikit-learn.org/stable/modules/cross_validation.html#). There are the splitter classes in [sklearn.model_selection](http://scikit-learn.org/stable/modules/classes.html#module-sklearn.model_selection)

- [x] K-Folds cross-validator
- [x] K-fold iterator variant with non-overlapping groups.
- [x] Stratified K-Folds cross-validator
- [x] Leave One Group Out cross-validator
- [x] Leave P Group(s) Out cross-validator
- [x] Leave-One-Out cross-validator
- [x] Leave-P-Out cross-validator
- [x] Random permutation cross-validator
- [x] Shuffle-Group(s)-Out cross-validation iterator
- [x] Stratified ShuffleSplit cross-validator
- [x] Predefined split cross-validator
- [x] Time Series cross-validator
